### DISCUSSION TITLE
"xrepo install" command shows "vs not found" even when msvc toolchain is not used

### DISCUSSION THREAD

**Discussion by vspefs** [xmake-io/xmake#5415] [Q&A] [answered]
While installing package 'vulkan-hpp' from the official GitHub xmake repo, after downloading the package, xmake shows "vs not found!" and exits.
My computer doesn't have MSVC installed because I use clang/llvm toolchain.

**Comment by star-hengxing** [comment, marked as answer]
Xmake package only support clang-cl/msvc toolchain on Windows, and only cross plat (`xmake f -p cross`) support any toolchain.
Currently, most of the pre-compiled llvm toolchain depends on msvc stl + ucrt.

```
$ clang -v
Target: x86_64-pc-windows-msvc
```
